coprocessor/fhevm-engine/stress-test-generator/src/utils.rs

```diff
@@ -272,6 +272,31 @@ pub async fn query_and_save_pks(
     Ok((pks, public_params))
 }
 
+pub async fn get_ciphertext_digests(
+    handle: &[u8],
+    pool: &sqlx::PgPool,
+) -> Result<(Vec<u8>, Vec<u8>), Box<dyn std::error::Error>> {
+    loop {
```

Comment:
Better restrict this loop using retries count limit so that it times out in case of S3 failures.